### PR TITLE
Feat/add task categories

### PR DESCRIPTION
**Pull request by GitGitarra**

### CODE REVIEW

File: src/app/components/task-category-list/task-category-list.component.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ import {TaskCategory} from '../../models/task-category';
   styleUrls: ['./task-category-list.component.css']
 })
 export class TaskCategoryListComponent implements OnInit {
-  taskCategories: TaskCategory[];
+  taskCategories: TaskCategory[] = [];
```
Comment (s-kania) [inline src/app/components/task-category-list/task-category-list.component.ts]: Please don't initialize array in definition.

File: src/app/components/task-category-list/task-category-list.component.ts [hunk 2]
```diff
@@ -8,7 +8,8 @@ import {TaskCategory} from '../../models/task-category';
...
+  newTaskCategory: TaskCategory = new TaskCategory();
```
Comment (s-kania) [inline src/app/components/task-category-list/task-category-list.component.ts]: Not used variable. Please remove it.

**Review by s-kania — CHANGES_REQUESTED** [review]
Good job! But please review my comments and do some changes before merging.

File: src/app/components/task-category-list/task-category-list.component.ts (hunk 2 above)
Comment (GitGitarra) [reply src/app/components/task-category-list/task-category-list.component.ts]: Thanks for good spot

**Review by s-kania — APPROVED** [review]
Ok.

**Pull request merged**